common/display/src/main/scala/uk/ac/wellcome/display/models/WorksIncludes.scala

```diff
@@ -1,44 +1,62 @@
 package uk.ac.wellcome.display.models
 
+sealed trait WorkInclude
```

Comment:
Do we still need this - this was for when we have V1 / V2 - if it has knock on effects we can change it later.